inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/util/SchemaChangeUtils.java

```diff
@@ -23,25 +23,31 @@
 import org.apache.inlong.sort.protocol.ddl.operations.Operation;
 import org.apache.inlong.sort.protocol.enums.SchemaChangePolicy;
 import org.apache.inlong.sort.protocol.enums.SchemaChangeType;
+import org.apache.inlong.sort.schema.ColumnSchema;
+import org.apache.inlong.sort.schema.TableChange;
 
 import com.google.common.base.Preconditions;
+import com.google.common.collect.Sets;
 
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Set;
 import java.util.StringJoiner;
+import java.util.stream.Collectors;
 
 /**
  * Schema-change Utils
  */
-public final class SchemaChangeUtils {
+public class SchemaChangeUtils {
```

Comment:
I don't think it is a good idea to remove final

Reply:
A class modified with final cannot be inherited.

IcebergSchemaChangeUtils inherits from SchemaChangeUtils. @yunqingmoswu